src/components/slider/slider.js

```diff
@@ -269,7 +269,9 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
     }
     function stepValidator(value) {
       if (angular.isNumber(value)) {
-        return Math.round(value / step) * step;
+        var formattedValue = (Math.round(value / step) * step);
+        // Format to 3 digits after the decimal point - fixes #2015.
+        return parseFloat(formattedValue.toFixed(3));
```

Comment:
You should use `return Math.round(formattedValue * 1000) / 1000`.
JSperf: http://jsperf.com/math-round-vs-tofixed2/5